### DISCUSSION TITLE
Support Navlungo as shipping carrier

### DISCUSSION THREAD

**Discussion by earslan9** [etsy/open-api#724] [Ideas]
hi @etsyachristensen 
As Navlungo, we are the tech based shipping and fulfillment startup widely used by Etsy sellers in Turkey.
We offer line haul shipping services also partner with DHL,FEDEX,UPS and our own consolidation shipping services and we have integration with ETSY for seller orders, our customers use this order data to create shipping label on navlungo.com. We ship appx 100K ETSY shipment from Turkey to all around the world.
We also work with Aftership. 
We would like to be a carrier partner in ETSY for Turkish sellers for their user experience.
![aaaa](https://user-images.githubusercontent.com/120509911/207421162-f978ca4c-e4cf-4ed0-87d4-e1bcb72e9fd1.png)

Can you please guide us on this.

Thank you.
Emrah

**Comment by kbasali** [comment]
@etsyachristensen here we are :)

**Comment by etsyachristensen** [comment]
@earslan9 I'm not entirely sure how this integration would work. Typically we will add new carriers that are supported in Aftership, but it sounds like you're not a carrier, but a reseller of carrier services. So I'm not sure how we'd add you unless you have your own tracking services and your own tracking url for all your shipments. This would allow the API users to select your company as the carrier, not the individual carriers you support.

**Reply by earslan9** [reply]
hello @etsyachristensen We use our own logistics services for ETSY (not known carriers like Fedex/UPS) and our own tracking systems with integrated aftership tracking systems. We widely known in Turkey for ETSY community. 
Any help us to reach related team? Many thanks

**Reply by etsyachristensen** [reply]
Ah ok. If that's the case, then we'd have to get the following information:

Example URLs for several shipments. We need to see what the url structure looks like such that we can direct users to a common url with a tracking code. For example: https://track.somecarrier.com/?tracking=xxxxxxxxx

Example of some tracking codes (they can be fake, but must match the typical formatting of a tracking code for the carrier).

**Reply by etsyachristensen** [reply]
To be clear, we will handle passing this information to the fulfillment team that actually implements adding carriers. We don't provide direct access to that team.

**Reply by earslan9** [reply]
hello @etsyachristensen 

here some  examples; 

can you please help us here, we known carrier in aftership as well, https://www.aftership.com/carriers/navlungo

https://navlungo.com/track?carrier=nvl&trackingNumber=NVL000011038759
https://navlungo.com/track?carrier=nvl&trackingNumber=NVL000011064629
https://navlungo.com/track?carrier=nvl&trackingNumber=NVL000010842402

**Reply by mk148a** [reply]
> To be clear, we will handle passing this information to the fulfillment team that actually implements adding carriers. We don't provide direct access to that team.

@etsyachristensen  Hello any update with this? Also we and most peoples(arround thousands peoples) make Etsy fullfillments and shipment with Navlungo with one day, so we "was" use Etsy integration with Navlungo for getting our order details easily and create shipment with one-click after send tracking number to Etsy. It was saved us for wasting time and make mistakes for writing wrong address. 

Also in your this update https://github.com/etsy/open-api/releases/tag/3.0.0-general-release-2024-06-03 (3.0.0 General Release 2024-06-03) break this...
If you are making a change specific to Turkey, you should at least evaluate the successes, breakthroughs, investments received, monthly cargo output numbers and end-user comments of the companies you will nominate as partners. It's a real shame that the integration of a large and reliable company like Navlungo "doesn't work" with this latest change you made.

I am add the video for Navlungo's Etsy integrations working here, I hope you understand better why you are victimizing us:
https://www.youtube.com/watch?v=yya2C_20d8k
